test/common/runtime/runtime_impl_test.cc

```diff
@@ -496,6 +497,11 @@ TEST_F(StaticLoaderImplTest, All) {
   testNewOverrides(*loader_, store_);
 }
 
+// Validate that Static Layer does not log as unsupported.
+TEST_F(StaticLoaderImplTest, NoUnsupportedStaticLog) {
+  EXPECT_LOG_NOT_CONTAINS("warning", "Skipping unsupported runtime layer", setup());
```

Comment:
While you are in here can you add a test for the warning also? Thank you!

/wait

Reply:
Hm, we actually shouldn't be able to get to this condition ever since there's a proto constraint that validates that we specify one of static/disk/admin/rtds. It'd run in to a proto validation error first. I suppose the warning would only come up if a new layered_runtime was added without support in LoaderImpl.

Reply:
Yeah if there is a constraint, just remove the warning instead and use NOT_REACHED

Reply:
I don't think this test is useful anymore. Can you revert these changes? Thank you!

/wait

Reply:
Sorry about that, should just be the NOT_REACHED case now (with proto validator).